superset-frontend/src/components/ButtonGroup/ButtonGroup.stories.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import Button from 'src/components/Button';
+import ButtonGroup from './index';
+
+export default {
+  title: 'ButtonGroup',
+  component: ButtonGroup,
+};
+
+export const ButtonGroupGallery = () => (
+  <>
+    <ButtonGroup css={{ marginBottom: 40 }}>
+      <Button buttonStyle="tertiary">Button 1</Button>
+    </ButtonGroup>
+    <ButtonGroup css={{ marginBottom: 40 }}>
+      <Button buttonStyle="tertiary">Button 1</Button>
+      <Button buttonStyle="tertiary">Button 2</Button>
+    </ButtonGroup>
+    <ButtonGroup>
+      <Button buttonStyle="tertiary">Button 1</Button>
+      <Button buttonStyle="tertiary">Button 2</Button>
+      <Button buttonStyle="tertiary">Button 3</Button>
+    </ButtonGroup>
+  </>
+);
+
+ButtonGroupGallery.story = {
+  parameters: {
+    actions: {
+      disabled: true,
+    },
+    controls: {
+      disabled: true,
+    },
+    knobs: {
+      disabled: true,
```

Comment:
Might be nice to have knobs for button size or button type, just to make sure the other flavors all work as expected... but not a big deal at all.

Reply:
I'm going to make it interactive with the option to change button size and style. Whenever I can I'm using controls instead of knobs, that's why I disabled the knobs panel.